src/blk/BlockDevice.cc

```diff
@@ -144,6 +141,11 @@ BlockDevice::device_type_from_name(const std::string& blk_dev_name)
   if (blk_dev_name == "hm_smr") {
     return block_device_t::hm_smr;
   }
+#endif
+#if defined(HAVE_LIBAIO) || defined(HAVE_POSIXAIO)
```

Comment:
i'd prefer to keep where it was, or create another commit to address this ordering thing as a cleanup change.

Reply:
yeah, can do in separate PR sometime, updated